flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaProducerMigrationOperatorTest.java

```diff
@@ -52,9 +51,6 @@ public String getOperatorSnapshotPath(FlinkVersion version) {
                 + "-snapshot";
     }
 
-    @Ignore
     @Override
-    public void writeSnapshot() throws Exception {
-        throw new UnsupportedOperationException();
-    }
+    public void writeSnapshot(FlinkVersion targetVersion) throws Exception {}
```

Comment:
It should be because that the base snapshots should only be available in 1.8 ~ 1.11, thus the there is no need to generate snapshots for this test in newer versions. The logic is also the same in the previous version.